docs/references/skyh5_memo.tex

```diff
@@ -0,0 +1,221 @@
+
+Many of the datasets in SkyH5 files have units associated with them (represented as \texttt{astropy Quantity} objects on the \texttt{SkyModel} object).
+The units are stored as attributes on the datasets with the name ``unit''.
+Datasets that derive from other \texttt{astropy} objects (e.g. \texttt{astropy Time,
+astropy EarthLocation, astropy Latitude, astropy Longitude})
+also have an ``object\_type'' attribute indicating the object type.
+
+In the discussion below, we discuss required and optional datasets in the
+various groups. We note in parenthesis the corresponding attribute of a \texttt{SkyModel}
+object. Note that in nearly all cases, the names are coincident, to make things
+as transparent as possible to the user.
+
+\section{Header}
+\label{sec:header}
+The \texttt{Header} group of the file contains the metadata necessary to interpret
+the data. We begin with the required parameters, then continue to optional
+ones. Unless otherwise noted, all datasets are scalars (i.e., not arrays). The
+precision of the data type is also not specified as part of the format, because
+in general the user is free to set it according to the desired use case (and
+HDF5 records the precision and endianness when generating datasets). When using
+the standard \texttt{h5py}-based implementation in \texttt{pyradiosky}, this typically
+results in 32-bit integers and double precision floating point numbers. Each
+entry in the list contains \textbf{(1)} the exact name of the dataset in the
+HDF5 file, in boldface, \textbf{(2)} the expected datatype of the dataset, in
+italics, \textbf{(3)} a brief description of the data, and \textbf{(4)} the name
+of the corresponding attribute on a \texttt{SkyModel} object.
+
+Note that string datatypes should be handled with care. See
+the Appendix in the UVH5 memo (\url{https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/docs/references/uvh5_memo.pdf})
+for appropriately defining them for interoperability between different HDF5 implementations.
+
+\subsection{Required Parameters}
+\label{sec:req_params}
+\begin{itemize}
+
+\item \textbf{component\_type}: \textit{string} The type of components in the sky model. The options are: ``healpix'' and ``point''.
+If component\_type is ``healpix'', the components are the pixels in a HEALPix map in units compatible with K or Jy/sr.
+If the component\_type is ``point'', the components are point-like sources, or point like components of extended sources,
+in units compatible with Jy or K sr. Some additional parameters are required depending on the component type. (\textit{component\_type})
+
+\item \textbf{Ncomponents}: \textit{int} The number of components in the sky model. This can be the number of individual
+compact sources, or it can include components of extended sources, or the number of pixels in a map. (\textit{Ncomponents})
+
+\item \textbf{spectral\_type}: \textit{string} This describes the type of spectral model for the components. The options are:
```

Comment:
This entry seems long enough that it's perhaps worthwhile to break the four spectral types into an enumerated sub-list.

Reply:
Yes, that makes it much cleaner.